htop.1.in

```diff
@@ -51,6 +51,9 @@ Show only the processes of a given user
 \fB\-U \-\-no-unicode\fR
 Do not use unicode but ASCII characters for graph meters
 .TP
+\fB\-m \-\-no-mouse\fR
```

Comment:
Shouldn't this be `-M` for consistency (disabled options in upper case)?

Reply:
I was also surprised, but it is `-m`. This PR only documents the current state. The option was introduced in b0e24cd.

Reply:
Can you make this -M as per #139 and update the pull request incl. the code change?
I can't really merge this one as #139 would be better :-)